src/utils/helpers.js

```diff
@@ -174,6 +174,15 @@ export function escapeRegExpChars(value) {
     // eslint-disable-next-line
     return value.replace(/[\-\[\]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&')
 }
+/**
+ * Remove accents/diacritics in a string in JavaScript
+ * https://stackoverflow.com/a/37511463
+ */
+export function removeDiacriticsFromString(value) {
+    if (!value) return value
+
+    return value.normalize('NFD').replace(/[\u0300-\u036f]/g, '')
```

Comment:
I'd say the best approach would be using `value.normalize("NFD").replace(/\p{Diacritic}/gu, "")` but looks like babel has problems with this regex https://github.com/babel/babel/issues/8951.